uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/JwtBearerGrantIT.java

```diff
@@ -98,11 +102,11 @@ public void testJwtBearerGrantForUnknownClient() {
 
```

Comment:
not sure - this is another case worth discussing. We generally try to minimize integration tests where something can be covered in unit tests. On other hand, for important use cases like this, a integration test could ensure that involved components are wired correctly?